hnsendpoint.go

```diff
@@ -192,18 +192,24 @@ func (endpoint *HNSEndpoint) ContainerHotDetach(containerID string) error {
 	return modifyNetworkEndpoint(containerID, endpoint.Id, Remove)
 }
 
-// ApplyACLPolicy applies Acl Policy on the Endpoint
-func (endpoint *HNSEndpoint) ApplyACLPolicy(policy *ACLPolicy) error {
+// ApplyPolicy applies a set of ACL Policies on the Endpoint

```

Comment:
Oh, this should be changed back to `ApplyACLPolicy` to keep Godoc happy

Reply:
Good catch, thanks. This is another hang-over from the vnext changes where this function has been renamed to ApplyPolicy in order to accept and apply a generic policy object, my bad for not catching it.